WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -1000,6 +1001,19 @@ - (void)logoutSimperiumAndResetNotifications
     [self.simperium signOutAndRemoveLocalData:YES completion:nil];
 }
 
+- (NSString *)notificationsBucketnameFromLaunchOptions:(NSDictionary *)launchOptions
```

Comment:
Slight nitpick, but the `name` should probably be capitalized. i.e. `notificationsBucketNameFromLaunchOptions`